climakitae/selectors.py

```diff
@@ -262,40 +251,29 @@ def view(self):
             color="k",
         )
         mpl_pane = pn.pane.Matplotlib(fig0, dpi=144)
+
+        def plot_subarea(boundary_dataset, extent):
+            ax.set_extent(extent, crs=xy)
+            subarea = boundary_dataset[boundary_dataset.index == shape_index]
+            df_ae = subarea.to_crs(crs_proj4)
+            df_ae.plot(ax=ax, color="b", zorder=2)
```

Comment:
This code plots `states` with a solid blue symbology like the other 2 datasets instead of a blue outline. Let me know if you want to keep these different.